cli/pkg/config/config.go

```diff
@@ -111,6 +115,15 @@ func InitConf(cfgFile string) *Config {
 		os.Exit(1)
 	}
 
+	// Set Tf Module Repo
+	err = setTfModuleRepo(conf.TFModuleRepo)
+	if err != nil {
+		log.Error(err)
```

Comment:
If `setTfModuleRepo` fails, do we want to exit fatal? right now we're just logging the error and continuing

Reply:
Added exit